plugin/externalPluginRoot.go

```diff
@@ -24,34 +25,36 @@ func newExternalPluginRoot(script string) *externalPluginRoot {
 
 // Init initializes the external plugin root
 func (r *externalPluginRoot) Init() error {
-	ctx := context.Background()
-	script := r.script
-
-	stdout, err := r.script.InvokeAndWait(ctx, "init")
+	// Give external plugins about five-seconds to finish their
+	// initialization
+	ctx, cancelFunc := context.WithTimeout(context.Background(), 5 * time.Second)
```

Comment:
Does this make sense? Should we go with a lower timeout?

Reply:
No, this seems fine. Low timeouts can be finicky.